Sass/style.css

```diff
@@ -0,0 +1,555 @@
+  padding-left: 10px;
+  margin-bottom: 20px;
+}
+
+.top-header li a {
+  color: white;
+}
+
+.courses a:hover {
+  transform: scale(1.15);
+  text-decoration-line: underline;
+}
+
+.courses {
+  color: white;
+  display: flex;
+  row-gap: 5px;
+  padding: 10px;
+  flex-direction: column;
+  justify-content: center;
+  background-color: rgba(255, 255, 255, 0.1);
+  box-sizing: border-box;
+  margin-right: 15px;
+  transform: scale(1.15);
+  align-items: center;
+  height: 240px;
+  max-width: 150px;
+}
+
+.courses:active {
+  border: #fff solid 1px;
+}
+
+.feedback-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 700px;
+  width: 100%;
+  padding: 10px;
+}
+
+.feedback-section h1 {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  color: black;
+  margin-bottom: 20px;
+}
+
+.feedback-section .inner-container {
+  display: grid;
+  width: 100%;
+  column-gap: 10px;
+  row-gap: 10px;
+  grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+
+  /* justify-content: space-around; */
+}
+
+.feedback-section .inner-container .cards {
+  display: flex;
+  width: 100%;
+  border-radius: 20px;
+  background-color: white;
+}
+
+.second-div {
+  display: flex;
+  flex-direction: column;
+  line-height: 1.5;
+
+  /* width: auto; */
+}
+
+.feedback-section .inner-container .cards .profile-name-div {
+  display: flex;
+  flex-direction: column;
+}
+
+.top-header {
+  position: sticky;
+  display: flex;
+  justify-content: flex-end;
+  column-gap: 24px;
+  padding-right: 192px;
+  color: white;
+  list-style-type: none;
+}
+
+.bg-dark {
+  background-color: #212529 !important;
+}
+
+.text-white {
+  color: #fff !important;
+}
+
+.py-1 {
+  padding-top: 0.25rem !important;
+  padding-bottom: 0.25rem !important;
+}
+
+/* .top-nav {
+  display: flex;
+   position: fixed;
+background-color: black;
+}
+.top-nav-menu {
+    list-style-type: none;
+      display: flex;
+  float: right;
+}
+.top-nav a img {
+  height: 35px;
+  width: 35px;
+} */
+
+/* footer .lets-connect a img {
+  height: 35px;
+  width: 35px;
+} */
+
+.bootcamp .bootcamp-div img {
+  margin-top: 10px;
+  width: 240px;
+  height: auto;
+}
+
+.feedback-section .inner-container .cards .profile-name-div img {
+  width: 80px;
+  height: 100px;
+  padding-right: 10px;
+}
+
+.bootcamp .bootcamp-div p {
+  display: block;
+  max-width: 335px;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+footer .inner-sub-div .footer-text p {
+  padding-right: 5px;
+  width: 100%;
+}
+
+.feedback-section .inner-container .cards p {
+  display: flex;
+  width: 100%;
+}
+
+.bootcamp {
+  display: flex;
+  height: auto;
+  background-color: #212529;
+  background-image: url(../images/bootcamp-background.png);
+  background-repeat: no-repeat;
+  background-size: 100% auto;
+  background-position: center bottom;
+  justify-content: center;
+  color: white;
+}
```

Comment:
Great design @degisew 🤝

- [ ] As the previous reviewers pointed out, You may customize your footer section according to the design provided.
- Given design
![given footer](https://user-images.githubusercontent.com/63917725/183267050-815e0491-03c3-4ac9-9c1d-ef4771bf559b.PNG)
- Your design
![your footer](https://user-images.githubusercontent.com/63917725/183267058-1d0d6928-940b-4e60-bab0-9912e5673616.PNG)